charity-data/getcharities.py

```diff
@@ -0,0 +1,80 @@
+from urllib import parse
+import requests
+import json
+
+base_url = 'https://api.data.charitynavigator.org/v2/Organizations?'
+
+# Parameters for request URL
+app_id = '80de6d65'
+app_key = '269363d5f87d1f1834f7b5ef5f9e4c3a'
+pageSize = '1000'
+rated = 'true'
+sizeRange = '3'
+sort = 'RATING'
+params = {'app_id': app_id, 'app_key': app_key, 'pageSize': pageSize,
+          'rated': rated, 'sizeRange': sizeRange, 'sort': sort}
+
+url = base_url + parse.urlencode(params)
+
+# Get JSON response body
+charity_nav_json = requests.get(url).json()
+
+# Counters for the number of charities with no valid donate link, the number
+# with no valid homepage link, and the number with no valid image link.
+num_no_valid_donate = 0
+num_no_valid_link = 0
+num_no_image = 0
+
+charities = []
+for charity in charity_nav_json:
+    # Create a charity JSON object
+    processed_charity = {}
+    
+    # Name
```

Comment:
you can remove such comments which are trivial and not explain much anyways

Reply:
Done.